### PR TITLE
Fix tests disk space in GitHub

### PR DESCRIPTION
**Pull request by afalhambra-hivemq**
We are failing to run integration tests as we fail to pull out container images when running out space in the GitHub runners.

For that, I just redistributed some of the failing tests such as `Extensions` tests for the compose GitHub workflow to reduce space disk usage and clean up containers and snapshots in the K3s container for all of the tests. This change is making the GitHub checks to not be fulfilled for this PR.

### CODE REVIEW

File: tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java [hunk 1]
```diff
@@ -188,4 +194,40 @@ protected boolean uninstallPlatformChart() {
...
+                if echo "$SNAPSHOT" | grep -q "Committed"; then \
+                echo "Removing snapshot $ID"; \
+                ctr snapshot rm $ID; \
+                fi; \
+                done""");
+        LOG.info("Removed unused snapshots:\n{}", pruneSnapshots.getStdout());
+        if (pruneSnapshots.getExitCode() != 0) {
+            LOG.warn("Error during removal of snapshots:\n{}", pruneSnapshots.getStderr());
+        }
+    }
```
Comment (afalhambra-hivemq) [inline tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java:232]: This code is already merged into both the `hivemq-platform-operator` and the `helm-charts` epic branches.
See https://github.com/hivemq/helm-charts/commit/490950089e39d7205fd0ea7a396173b24887eb12

File: tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/compose/HelmBridgeExtensionIT.java [hunk 2]
```diff
@@ -21,6 +21,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @Tag("Extensions")
```
Comment (LukasBrand) [inline tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/compose/HelmBridgeExtensionIT.java:23]: Is this still used?

File: tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java (hunk 1 above)
Comment (LukasBrand) [reply tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java:232]: 😄 Does not make it prettier

**Review by LukasBrand — APPROVED** [review]

File: tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java (hunk 1 above)
Comment (afalhambra-hivemq) [reply tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java:232]: Definitely, extremely ugly! 🙈

**Pull request merged**